app/javascript/entrypoints/application.js

```diff
@@ -45,6 +45,7 @@ document.addEventListener('DOMContentLoaded', () => {
 
 /* eslint-disable no-console */
 /* eslint-disable no-undef */
+/* eslint-disable func-names */
```

Comment:
This is unrelated to this PR but ESLint started complaining about it so I suppressed the warning.